docs/reference/ingest/apis/enrich/execute-enrich-policy.asciidoc

```diff
@@ -93,8 +94,13 @@ The previous enrich index will deleted with a delayed maintenance job.
 By default this is done every 15 minutes. 
 // end::update-enrich-index[]
 
-Because this API request performs several operations,
-it may take a while to return a response.
+By default, this API is synchronous: it returns when a policy has been executed.
+Because executing a policy performs several operations, it may take a while to
+return a response, especially when the source indices are large. This can lead
+to timeouts. To prevent those, set the `wait_for_completion` parameter to
+`false`. This runs the request asynchronously in the background, and returns a
+task ID. You can use the task ID to manage the request with the <<tasks,task
+management API>>.
```

Comment:
```suggestion
By default, this API is synchronous: It returns when a policy has been executed.
Because executing a policy performs several operations, it may take a while to
return a response, especially when the source indices are large. This can lead
to timeouts. To prevent timeouts, set the `wait_for_completion` parameter to
`false`. This runs the request asynchronously in the background, and returns a
task ID. You can use the task ID to manage the request with the <<tasks,task
management API>>.
```